src/Cli/dotnet/commands/dotnet-test/TestingPlatformCommand.cs

```diff
@@ -46,17 +46,24 @@ public int Run(ParseResult parseResult)
                 {
                     testApp.HelpRequested += OnHelpRequested;
                     testApp.ErrorReceived += OnErrorReceived;
+                    testApp.TestProcessExited += OnTestProcessExited;
 
-                    await testApp.RunHelpAsync();
+                    int runHelpResult = await testApp.RunHelpAsync();
```

Comment:
we can followup on this one..but we need to OR all the results and use it to return in line 99 != 0 if something went wrong:

* tests failed
* test crashed etc...